tooling/bundler/src/bundle/windows/templates/main.wxs

```diff
@@ -334,6 +337,10 @@
         </InstallExecuteSequence>
         {{/if}}
 
+        <InstallExecuteSequence>
+          <Custom Action="LaunchApplication" After="InstallFinalize"><![CDATA[NOT(AUTOLAUNCHAPP="True")]]> AND NOT Installed</Custom>
```

Comment:
Can this be changed to also accept a list of arguments?

Reply:
Do you mean to be able to re-apply the CLI flags the initial app instance was started with?

It should be possible, I'll take a look if that's what you meant.

Reply:
Yeah that's what I meant

Reply:
whoever invented spaces can go straight to hell

Reply:
I can't believe how long it took me to figure out the code for the updater plugin ffs (in handsight it was easy but i never would have guessed to escape `"` with `""` in arguments already encapsulated with `"`...)

https://github.com/tauri-apps/plugins-workspace/pull/1498